### PR TITLE
Work on typography

### LINKED ISSUE #20: Typography
This will require styling the links, ```<p></p>``` tags, and headers. (Remember that we decided to go with downcase headers and links) 

I believe we decided to do ```text-decoration: none;``` for the links so go ahead and do that. Don't do anything with the colors yet. That will be for another issue.

With the ```<p></p>``` tags, I was thinking we should have them centered on the page. If you try that and it doesn't look good, that's fine. Feel free to give suggestions or ask any questions you have. 

The **asphalt** header seems too small to me:
screen shot 2015-04-29 at 11 28 06 am

==================
I think we should make it a little bit bigger but leave the subheader the same. Again, suggestions are great. We need to make sure we collaborate as much as possible so we don't conflict.

### PR DESCRIPTION
**Pull request by matthewberkompas**
Change size of headers and paragraphs as well as style links. Add projects button to home page. Closes #20 

I got a pretty good start on this issue. As we go on, we should be able to fill it in a little bit more once we get the content finalized. Here are a few screenshots: 

I added a button which leads to projects. I tried a few things but wasn't able to change the color of the text inside it yet though. Also, I changed the size of the main header.
screen shot 2015-05-04 at 12 28 20 pm
============
I put in a couple paragraphs just to fill the space for now. I was thinking we might want to put some pictures or something like that in here eventually so that will help fill it out.
screen shot 2015-05-04 at 12 29 14 pm
============
I removed the text decoration from the links.
screen shot 2015-05-04 at 12 30 12 pm

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -16,6 +16,77 @@
...
  */
 
+
+/* Typography */
+
 .pure-menu .pure-menu-heading {
   text-transform: lowercase;
 }
+
+#home-title {
```
Comment (danielberkompas) [inline app/assets/stylesheets/application.css]: It's best not to use a lot of ids in your CSS. Here, you just want a bigger header, so make a class `large-header` or something.

File: app/views/projects/index.html.erb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <% provide(:title, 'Projects') %>
 <% provide(:header) do %>
-  <h1>project ideas</h1>
+  <h1 id="top-header">project ideas</h1>
```
Comment (danielberkompas) [inline app/views/projects/index.html.erb]: Is this `#top-header` style something that couldn't be applied to all `<h1>`?

File: app/assets/stylesheets/application.css (hunk 1 above)
Comment (matthewberkompas) [inline app/assets/stylesheets/application.css]: The header wouldn't respond to `class="home-title"` but when I used an id, it did what I wanted. I'll see if I can get it to work with a class though.

Comment (matthewberkompas) [inline app/assets/stylesheets/application.css]: I pushed up my branch. It ended up that I didn't need the other id so I got rid of that but I couldn't get the `#home-title` to work as `.home-title`.

Comment (danielberkompas) [inline app/assets/stylesheets/application.css]: I don’t think it should be .home-title. I think it should be .large-title. Try adding the !important tag after the font size like this:

```
font-size: 150px !important;
```

Comment (matthewberkompas) [inline app/assets/stylesheets/application.css]: That worked!

Comment (danielberkompas) [inline app/assets/stylesheets/application.css]: Great, but I'm still not seeing it changed in the diff.

**Pull request merged**